include/Packet.hpp

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+#include <cstdint>
+
+class Packet {
+public:
+    explicit Packet(std::uint64_t sending_time_ns);
+    Packet() = default;
+    ~Packet() = default;
+
+    /**
+     * @brief Packet sending time in nanoseconds
+     */
+    std::uint64_t sending_time_ns;
```

Comment:
> std::uint64_t sending_time_ns;

The variable name seems self-explanatory, so no need to accompany it with comment `Packet sending time in nanoseconds`.